test/integration/cdi/example-driver/app/kubeletplugin.go

```diff
@@ -98,6 +93,7 @@ func (ex *examplePlugin) stop() {
 // gets written again).
 func (ex *examplePlugin) NodePrepareResource(ctx context.Context, req *drapbv1.NodePrepareResourceRequest) (*drapbv1.NodePrepareResourceResponse, error) {
 	logger := klog.FromContext(ctx)
+	deviceName := "claim-" + req.ClaimUid
```

Comment:
@klihub: is this correct?

I tried it and it fails to start the pod:
```
Error: CDI device injection failed: unresolvable CDI devices example-driver.cdi.k8s.io/claim-31220e97-8655-4cc8-94e2-7f68e57273f1=claim-31220e97-8655-4cc8-94e2-7f68e57273f1
```

The `=claim-31220e97-8655-4cc8-94e2-7f68e57273f1` suffix looks odd, but that is what `cdiSpec.GetDevice(deviceName).GetQualifiedName()` returned.

Here's the generated file:

```
$ cat /var/run/cdi/example-driver.cdi.k8s.io-31220e97-8655-4cc8-94e2-7f68e57273f1.json 
{"cdiVersion":"0.2.0","kind":"example-driver.cdi.k8s.io/claim-31220e97-8655-4cc8-94e2-7f68e57273f1","devices":[{"name":"claim-31220e97-8655-4cc8-94e2-7f68e57273f1","containerEdits":{"env":["user_a=b"]}}],"containerEdits":{}}
```

Reply:
Just having a different file with UID in `/var/run/cdi/` causes the same failure. After removing all files and using `deviceName := "claim-example" it works again.

Could it be that a valid (?) kind still causes problems?

Reply:
This is with cri-o 1.24.2.

Reply:
Also occurs when using kind=example-driver.cdi.k8s.io/test:

```
  Warning  Failed            7s (x3 over 9s)  kubelet            Error: CDI device injection failed: unresolvable CDI devices example-driver.cdi.k8s.io/test=claim-06ad4536-9d3f-4caf-8791-f74732d58206
```

@bart0sh : one error message in kubelet has a bug:
```
I0811 08:13:58.532257   11916 manager.go:265] GetCDIAnnotations: claim [{cdi.k8s.io/example-driver.cdi.k8s.io_06ad4536-9d3f-4caf-8791-f74732d58206 example-driver.cdi.k8s.io/test=claim-06ad4536-9d3f-4caf-8791-f74732d58206}]: add resource annotations: %!v(MISSING)
```

Reply:
Never mind. The root cause turned out to be a missing `/etc/cdi`: https://github.com/container-orchestrated-devices/container-device-interface/issues/78